README.md

```diff
@@ -35,9 +35,23 @@ check the `examples` subdirectory.
 
 ## Install
 
+Install it via go get
 ```
 go get github.com/owulveryck/onnx-go
 ```
+Or add it in go.mod
+
+```
+module myproject.org/package
+
+go 1.12
```

Comment:
is go 1.12 mandatory? If so, we should write it?

Reply:
The code is tested only with go 1.12. Do we support other golang version ?

Reply:
As you told me offline, it would be nice to support also go 1.11 as it's the default version shipped with Debian.

Reply:
I tested with go 1.11 and gomodules are disable out of the box and the support is not great.